crates/turborepo-lockfiles/src/pnpm/data.rs

```diff
@@ -217,15 +217,9 @@ impl PnpmLockfile {
             importer.dependencies.find_resolution(name)
         else {
             // Check if the specifier is already an exact version
-            return match self.get_packages(&self.format_key(name, specifier)) {
-                Some(_) => Ok(Some(specifier)),
-                None => Err(Error::MissingResolvedVersion {
```

Comment:
This is the only instance of `MissingResolvedVersion` in the codebase, can you `rm` that too?